pkg/util/grpcclient/grpcclient.go

```diff
@@ -29,8 +29,9 @@ type Config struct {
 	BackoffOnRatelimits bool           `yaml:"backoff_on_ratelimits"`
 	BackoffConfig       backoff.Config `yaml:"backoff_config"`
 
-	TLSEnabled bool             `yaml:"tls_enabled"`
-	TLS        tls.ClientConfig `yaml:",inline"`
+	TLSEnabled         bool             `yaml:"tls_enabled"`
+	TLS                tls.ClientConfig `yaml:",inline"`
+	EnableSignRequests bool             `yaml:"-"`
```

Comment:
nit: for consistency I think `SignRequestsEnabled` is a more consistent naming convention as above :)